go/service/ui_router.go

```diff
@@ -155,3 +155,20 @@ func (u *UIRouter) GetRekeyUI() (keybase1.RekeyUIInterface, int, error) {
 	}
 	return ret, sessionID, nil
 }
+
+func (u *UIRouter) GetRekeyUINoSessionID() (keybase1.RekeyUIInterface, error) {
```

Comment:
The other one (GetRekeyUI) requests a session id from electron.  This is for the one-off interactions where CLI would like to specify 0 for session id so that electron knows that it is a one-off UI interaction.